src/containers/audio-rewards-page/WalletModal.tsx

```diff
@@ -186,6 +204,8 @@ const ModalContent = ({
   // TODO: user models need to have wallets
   const wallet = account.wallet as WalletAddress
 
+  const solWallet = account.userBank!
```

Comment:
let's avoid using forced unwrapping, it defeats the purpose of TS's nice optional handling. Lets either check this and return null here, or more ideally fix the account model if we know that the userBank will never be null

Reply:
I can write the null case, but this is the common pattern when we know it's the "current" account - it should be present but will always be optional - https://github.com/AudiusProject/audius-client/blob/c6224146c9d52796f1c96d2c0e631610148ee6cd/src/common/models/User.ts#L39-L52

Reply:
could we just return null like above if we don't have it?